tools/api-goldens/test_api_report.ts

```diff
@@ -37,13 +35,51 @@ import { resolveTypePackages } from './interop_module_mappings.js';
  * */
 const _origFetchAstModuleExportInfo = ExportAnalyzer.prototype.fetchAstModuleExportInfo;
 
+/** Angular static fields that are skipped */
+const _angularStaticFields = ['ɵcmp', 'ɵdir', 'ɵfac', 'ɵmod', 'ɵinj', 'ɵprov'];
```

Comment:
```suggestion
const _angularStaticFields = [ 'ɵdir', 'ɵfac', 'ɵmod', 'ɵinj', 'ɵprov'];
```

I think we should keep `ɵcmp`.  This has relevant information, like selector and aliases which we need.

Reply:
@spike-rabbit @dr-itz Yeah, I guess we need to keep the `ecmp` also to generate our docs from it. Although that could maybe be done in a dynamic way and would not have to be commited to the repo.

Reply:
Well, I _completely_ disagree here. `ɵcmp` contains Angular meta data, but repeats inputs and outputs so is very noisy with every change.

It's about the added value of the API goldens in general. Currently IMO the added value is ZERO if not negative. I.e. it adds a lot of noise to MRs and doesn't make review easier. So what is the added value? If anything, the API goldens should make it easier to spot accidental breaking changes, no? And for it to be easier it simply cannot be noisy. Because frankly I see all API changes in the changed code _anyway_. And it's not that hard to spot breaking changes either. So for me, right now the API goldens are completely backwards and annoying so I'd rather see them gone. Or fixed to be useful.

If you want to generate docs from Angular related stuff like selectors, then parsing output from api-extractor is a way, but certainly not the generated API reports which are Markdown files. So filtering the Angular stuff in the `Collector` seems to be the wrong place and it should be purely done in the `ApiReportGenerator`. It's actually easier there but a tad uglier since it needs patching a private static method. I did that in the last commit.